packages/test/data/monad002.ly

```diff
@@ -135,7 +136,7 @@ The following functions should be defined for all parsers.
 This is the overall type of the parser.
 
 > type Parse = P Exp
-> calc :: Parse
+> -- calc :: Parse
```

Comment:
Why this change?

Reply:
The explicit type annotation did conflict with my rad-parser, which would add a type annotation itself when called with `--types`. I'm not sure if that's reason enough to really remove it, but since the type annotation isn't required for the test to succeed under normal `happy`, I did it.

Reply:
```suggestion
> calc_test_sig :: Parse
> calc_test_sig = calc

(We use a fresh name because `calc` might already have a sig depending on the backend.)
```